docs/Buttons/GetCapabilities/assets/GetCapabilities.txt

```diff
@@ -0,0 +1,36 @@
+participant App
+participant SDL
+participant HMI
+
+SDL<-HMI:BC.OnReady
```

Comment:
@KhrystynaDubovyk 
It is due to specifics of the tool used for creating a diagram that the text version reflects the sequence this way. I agree that the position of actors in source code might be misleading, however, our main focus is the diagram image and it reflects the correct sequence. 
Do you recommend to rework all the diagrams source codes to address the actors' sequence issue?

Reply:
I've been using the same tool. I had no issues writing `HMI ->SDL:` instead of `SDL<-HMI:`.
If it is possible, please, the next time, put the actor first. 
I don't only use the _images_, but also **_source files_** (for commenting and editing). So while editing I'll be checking image -> go to the source file and -> look for the actor at a certain step.